### PR TITLE
[HO] `cvd start` using canonical configs.

### PR DESCRIPTION
**Pull request by ser-io**

### CODE REVIEW

File: frontend/src/liboperator/api/v1/messages.go [hunk 1]
```diff
@@ -154,6 +159,23 @@ type CVD struct {
 	Displays []string `json:"displays"`
 }
 
+type Instance struct {
```
Comment (jemoreira) [inline frontend/src/liboperator/api/v1/messages.go:162]: Dont' try to parse the contents of the config file from the orchestrators (yet). Just pass them as is to the `cvd load/start` command.

The reason for this is that what goes in those files is still a bit unstable and this project doesn't really have a need to parse that yet. So treating those files as black boxes will save us a lot of time.

**Review by jemoreira — CHANGES_REQUESTED** [review]

File: frontend/src/liboperator/api/v1/messages.go (hunk 1 above)
Comment (ser-io) [reply frontend/src/liboperator/api/v1/messages.go:164]: I understand. Done.

Comment (jemoreira) [reply frontend/src/liboperator/api/v1/messages.go:164]: Looks like you could delete these structures now

Comment (ser-io) [reply frontend/src/liboperator/api/v1/messages.go]: I thought I did it, good catch. Thanks.

**Review by jemoreira — APPROVED** [review]

**Pull request merged**